### ISSUE TITLE
crosslinks need to be provided as comma separated, not space

### ISSUE THREAD

**Issue by gieses** [lhatsk/AlphaLink#3]
Hi Kolja,

a small hickup from the readme: it says crosslinks need to be space separated but the code does 

` links = np.loadtxt(crosslink_csv,delimiter=',')`.

Cheers,

**Comment by lhatsk** [comment]
Indeed, sorry! Thanks, Sven for letting me know. I changed it to be actually space-separated.
